integration_tests/integration/bookedtoday/arrivals/existingRecord.spec.ts

```diff
@@ -18,31 +18,31 @@ context('Confirm Arrival', () => {
   it('PER record should contain a full set of correctly formatted move data', () => {
     cy.task('stubExpectedArrival', expectedArrival)
     cy.signIn()
-    const confirmArrivalPage = ConfirmArrivalPage.goTo(expectedArrival.id)
+    const existingRecordPage = ExistingRecordPage.goTo(expectedArrival.id)
```

Comment:
split this ConfirmArrivalPage into two - `ExistingRecordPage` and `NoExistingRecordPage`. 


Trying to model as one page was a bit messy